usaspending_api/accounts/serializers.py

```diff
@@ -115,3 +115,10 @@ class Meta:
                 "kwargs": {"read_only": True}
             }
         }
+
+
+class ObjectClassFinancialSpendingSerializer(serializers.Serializer):
+
+    major_object_class_code = serializers.CharField()
+    major_object_class_name = serializers.CharField()
+    obligated_amount = serializers.DecimalField(21, 2)
```

Comment:
@nmonga91 Wanted to pull you in here...I started a pattern of serializing amounts as `DecimalField` instead of characters. It doesn't seem to affect the actual output, which is still rendered in quotes. What are your feelings on this?